tzrec/models/rank_model.py

```diff
@@ -183,6 +185,9 @@ def _loss_impl(
     ) -> Dict[str, torch.Tensor]:
```

Comment:
The sample_weight_name parameter, similar to label_name, needs to be an input for the _loss_impl function. In a multi-task ranking model, it's essential to specify distinct sample weights for each loss function.

Reply:
fixed